content/en/docs/concepts/security/_index.md

```diff
@@ -3,4 +3,241 @@ title: "Security"
 weight: 85
 description: >
   Concepts for keeping your cloud-native workload secure.
+simple_list: true
 ---
+
+This section of the Kubernetes documentation aims to help you learn to run
+workloads more securely, and about the essential aspects of keeping a
+Kubernetes cluster secure.
+
+Kubernetes is based on a cloud-native architecture, and draws on advice from the
+{{< glossary_tooltip text="CNCF" term_id="cncf" >}} about good practice for
+cloud native information security.
+
+## Cloud native information security
+
+The CNCF [white paper](https://www.cncf.io/wp-content/uploads/2022/06/CNCF_cloud-native-security-whitepaper-May2022-v2.pdf)
+on cloud native security defines security controls and practices that are
+appropriate to different _lifecycle phases_.
+
+## _Develop_ lifecycle phase {#lifecycle-phase-develop}
+
+- Ensure the integrity of development environments.
+- Design applications following good practice for information security,
+  appropriate for your context.
+- Consider end user security as part of solution design.
+
+To achieve this, you can:
+
+1. Adopt an architecture, such as [zero trust](https://glossary.cncf.io/zero-trust-architecture/),
+   that minimises attack surfaces, even for internal threats.
+1. Define a code review process that considers security concerns.
+1. Build a _threat model_ of your system or application that identifies
+   trust boundaries. Use that to model to identify risks and to help find
+   ways to treat those risks.
+1. Incorporate advanced security automation, such as _fuzzing_ and
+   [security chaos engineering](https://glossary.cncf.io/security-chaos-engineering/),
+   where it's justified.
+
+## _Distribute_ lifecycle phase {#lifecycle-phase-distribute}
+
+- Ensure the security of the supply chain for container images you execute.
+- Ensure the security of the supply chain for the cluster and other components
+  that execute your application. An example of another component might be an
+  external database that your cloud-native application uses for persistence.
+
+To achieve this, you can:
+
+1. Scan container images and other artifacts for known vulnerabilities.
+1. Ensure that software distribution uses encryption in transit, with
+   a chain of trust for the software source.
+1. Adopt and follow processes to update dependencies when updates are
+   available, especially in response to security announcements.
+1. Use validation mechanisms such as digital certificates for supply
+   chain assurance.
+1. Subscribe to feeds and other mechanisms to alert you to security
+   risks.
+1. Restrict access to artifacts. Place container images in a
+   [private registry](/docs/concepts/containers/images/#using-a-private-registry)
+   that only allows authorized clients to pull images.
+
+## _Deploy_ lifecycle phase {#lifecycle-phase-deploy}
+
+Ensure appropriate restrictions on what can be deployed, who can deploy it,
+and where it can be deployed to.
```

Comment:
```suggestion
and where it can be deployed.
```

Reply:
I (still) think “where is can be deployed to” is how people who speak English would say it.

Reply:
+1 to what Tim said.